docs/buildah-build.1.md

```diff
@@ -128,6 +128,12 @@ buildah build -t test --layers --cache-to registry/myrepo/cache --cache-from reg
 
 Note: `--cache-to` option is ignored unless `--layers` is specified.
 
+**--cache-ttl**
+
+Accepts time duration and will only consider cached intermediate images created under this specified duration.
+For example if `--cache-ttl=1h` is specified buildah will only consider intermediate cache images which are created
+under the duration of one hour and intermediate cache images outside this duration will be ignored.
```

Comment:
```suggestion
Limit use of cached images to only consider images with create timestamps less than *duration* ago.
```
"create timestamp" is the phrasing we use to describe the "--timestamp" option, which someone is sure to try to use in combination with this one at some point.

Reply:
Amended but left the example line just in case someone wants to look at the example but can drop if needed.